Gemfile

```diff
@@ -1,10 +1,15 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.6'
```

Comment:
This change pulls in the specific rails components we need below ('actionmailer', 'activerecord', etc.), leaving out components for things like view rendering which would be included by the full rails gem. It's not vital that we remove these dependencies, but I kinda like being a bit more minimal.